Packs/GoogleWebRiskAPI/Integrations/GoogleWebRiskV1/GoogleWebRiskV1.py

```diff
@@ -0,0 +1,40 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+import requests
+from ast import literal_eval
+
+Apikey = demisto.params().get('APIkey')
+ThreatCategories = demisto.params().get('ThreatCategories')
+search = ''
+
+for item in ThreatCategories:
+    cat = "&threatTypes="+item
+    search = search + cat
```

Comment:
Im cant see where you use the search variable.
I see the usage of cat in line 18.